src/main/drivers/serial_resource.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/*
+  #defines used for serial port resource access (pin/dma/inversion)
+  target/serial_post.h normalizes enabled port definitions, values are just remaned here
```

Comment:
```suggestion
  target/serial_post.h normalizes enabled port definitions, values are just remained here
```

Reply:
Thanks!
Actually  `renamed` is correct.